packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -223,12 +224,73 @@ void updateLocalProperties({String projectPath, BuildInfo buildInfo}) {
     settings.writeContents(localProperties);
 }
 
+void findAndReplaceVersionProperties({BuildInfo buildInfo}) {
+
+  // early return, if nothing has to be changed
+  if (buildInfo?.buildNumber == null && buildInfo?.buildName == null) {
+    return;
+  }
+
+  final File appGradle = fs.file('android/app/build.gradle');
```

Comment:
You have to use fs.path.join() to make this Windows-friendly (and on the next line)

Reply:
I wasn't sure whether I have to do this or not.
I guess line 62 needs to be fixed too?